kbs/src/attestation/intel_trust_authority/mod.rs

```diff
@@ -59,36 +71,63 @@ pub struct IntelTrustAuthority {
 #[async_trait]
 impl Attest for IntelTrustAuthority {
     async fn verify(&self, tee: Tee, nonce: &str, attestation: &str) -> Result<String> {
-        if tee != Tee::Tdx && tee != Tee::Sgx {
-            bail!("Intel Trust Authority: TEE {tee:?} is not supported.");
-        }
         // get quote
         let attestation = serde_json::from_str::<Attestation>(attestation)
             .context("Failed to deserialize Attestation request")?;
-        let evidence =
-            serde_json::from_value::<IntelTrustAuthorityTeeEvidence>(attestation.tee_evidence)
-                .context("Failed to deserialize TEE Evidence")?;
 
         let runtime_data = json!({
             "tee-pubkey": attestation.tee_pubkey,
             "nonce": nonce,
         })
         .to_string();
 
-        // construct attest request data
-        let req_data = AttestReqData {
-            quote: evidence.quote,
-            runtime_data: STANDARD.encode(runtime_data),
+        // construct attest request data and attestation url
+        let (req_data, att_url) = match tee {
+            Tee::AzTdxVtpm => {
+                let att_url = format!("{}{}/azure/tdxvm", &self.config.base_url, BASE_AS_ADDR);
```

Comment:
I thought about this a bit more and perhaps the az path could be a const too:
```suggestion
                let att_url = format!("{}{AZURE_TDXVM_ADDR}", &self.config.base_url);
```